rocksdb_admin/admin_handler.cpp

```diff
@@ -483,6 +483,9 @@ AdminHandler::AdminHandler(
       LOG(INFO) << "Stopping DB deletioin thread ...";
     });
   }
+
+  const int upload_MBps = 600; // fixed upper bw limit for now.
```

Comment:
As @rajathprasad suggest, besides `s3_download_limit_mb` gflag in `admin_handler.cpp`, we could add another one for backup instead of hardcoded 600.

Reply:
Done, added gflags setting with 600 as default.